dev-vcs/tig/tig-2.2.1.recipe

```diff
@@ -46,11 +55,23 @@ BUILD()
 {
 	libtoolize --force --copy --install
 	./autogen.sh
-	runConfigure ./configure
+	runConfigure --omit-dirs binDir ./configure --bindir=$commandBinDir
 	make
 }
 
 INSTALL()
 {
 	make install
+
+	# temporary solution because of xmlto deps failing
```

Comment:
That's the point.  Maybe **xmlto** works in general, but here it fails on some docbook validation.  It's not needed at all, as manuals in man format are delivered in the source package. This doesn't seem to me as a blocker.